### PR TITLE
update to 2.7.0

### PR DESCRIPTION
**Pull request by Arishamays1**
Checklist
* [ ] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -46,11 +46,17 @@ test:
 
 about:
   home: http://hdfscli.readthedocs.org
```
Comment (cbouss) [inline recipe/meta.yaml:48]: To update:
doc_url: https://hdfscli.readthedocs.io/
dev_url: https://github.com/mtth/hdfs/
license_family: MIT

**Review by cbouss — COMMENTED** [review]
Please update the about section

**Review by ELundby45 — APPROVED** [review]
lgtm

**Review by cbouss — APPROVED** [review]

**Review by skupr-anaconda — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (Arishamays1) [reply recipe/meta.yaml:48]: Thanks! Made those changes

**Pull request merged**